vllm/model_executor/layers/quantization/compressed_tensors/compressed_tensors.py

```diff
@@ -139,7 +176,8 @@ def apply(self,
         """
         Use the output of create_weights and the CompressedTensorsScheme 
         associated with the layer to apply the forward pass with the 
-        layer input.
+        layer input.  See LinearMethodBase for param details
```

Comment:
nit - make PR cleaner (dont touch this line)

Reply:
The comment was expanded